### DISCUSSION TITLE
WindowBuilder palette context menu problem

### DISCUSSION THREAD

**Discussion by vic-nunes** [eclipse/windowbuilder#319] [Q&A] [answered]
Hello,

I've tried several combinations of different versions of eclipse, and java, and WB, and i cannot make context menu to appear with full contents, it only has 2 options:
Layout and Settings.
I need to manage the palette to add extra items ( swingx ) and accordingly to the user manual, it should have a very different context menu... ( https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.wb.doc.user%2Fhtml%2Findex.html )

My Menu:
![image](https://user-images.githubusercontent.com/113114694/189130569-dbb71347-d206-4015-be16-69961ad8654d.png)

Supposed Menu:
![image](https://user-images.githubusercontent.com/113114694/189131012-0d97886b-8244-4300-a593-e3fac9345d3f.png)

i've searched and tried a lot of things i could find related in the internet, but nothing works.

note: Running on Windows 10

thanks in advance for your support!
Vitor Nunes

**Comment by ahmedamoharram** [comment]
Having the same issue now, only solution is to install windowbuilder v1.9.7 and use java 11 to launch eclipse (in the eclipse.ini file)

**Reply by wimjongman** [reply]
@MarcelS4 another one

**Reply by MarcelduPreez** [reply]
@wimjongman thanks I'll take a look at what happened here

**Reply by MarcelduPreez** [reply]
@wimjongman  please see PR #337, fixes the issue described here thanks

**Comment by suhasdesai-ecl** [comment]
Hello, I have the same problem as Vic-nunes. I am using windowbuilder version 1.9.8 and eclipse 2022-09. Was your problem solved? If so, then can you tell me how. I am a novice Java developer using Eclipse and I need to import JCalendar Jar.

Thanks

**Comment by wimjongman** [comment, marked as answer]
This issue has been resolved. Please update to the latest release

**Comment by suhasdesai-ecl** [comment]
Hello,
That was quick. It worked. 
Appreciate your time and help.
Suhas
    On Monday, October 24, 2022 at 05:44:50 AM CDT, Wim Jongman ***@***.***> wrote:  
 
 


This issue has been resolved. Please update to the latest release
